app/src/main/java/eu/kanade/presentation/updates/UpcomingItem.kt

```diff
@@ -0,0 +1,53 @@
+package eu.kanade.presentation.updates
+
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxHeight
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.text.style.TextOverflow
+import androidx.compose.ui.unit.dp
+import eu.kanade.presentation.manga.components.MangaCover
+import tachiyomi.domain.manga.model.Manga
+import tachiyomi.domain.manga.model.asMangaCover
+import tachiyomi.presentation.core.components.material.padding
+
+private val UpcomingItemHeight = 96.dp
+
+@Composable
+fun UpcomingItem(
+    upcoming: Manga,
+    modifier: Modifier = Modifier,
+    onClick: (manga: Manga) -> Unit = {},
```

Comment:
Pull out the `onClick` logic
```suggestion
    upcoming: Manga,
    onClick: () -> Unit,
    modifier: Modifier = Modifier,
```